drivers/spi/spi_ll_stm32.c

```diff
@@ -364,6 +400,7 @@ static void spi_stm32_shift_m(SPI_TypeDef *spi, struct spi_stm32_data *data)
 	}
 
 	spi_stm32_read_next_frame(spi, data);
+#endif
```

Comment:
Because that would modify logic that will be used for other SPI controllers apart from the H7's. I don't have access to these platforms, so I can't test these changes on them; I want to only modify logic I can test, as much as possible.

Reply:
As you want, it clutters the code a bit though.